libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Http;
+using BotMiddleware = Microsoft.Bot.Builder.Middleware;
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core
+{
+    public class BotFrameworkOptions
+    {
+        private readonly List<BotMiddleware.IMiddleware> _middleware;
+
+        public BotFrameworkOptions()
+        {
+            _middleware = new List<BotMiddleware.IMiddleware>();
+
+            RouteBaseUrl = "/bot";
+        }
+
+        public PathString RouteBaseUrl { get; set; }
+        public string ApplicationId { get; set; }
+        public string ApplicationPassword { get; set; }
```

Comment:
Normally we use MicrosoftAppId and MicrosoftAppPassword (I think)...we should use same names as rest of system